packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -57,26 +64,34 @@ export const GetDataCopySingleton = ({ value, onAction }: GetDataTrioProps) => {
 export const GetDataCopySingletonInline = ({
   value,
   onAction,
+  setTooltipVisible,
 }: GetDataTrioProps) => {
   return (
-    <FtlMsg id="get-data-trio-copy-2" attrs={{ title: true, ariaLabel: true }}>
-      <button
-        title="Copy"
-        type="button"
-        onClick={async () => {
-          const copyValue = Array.isArray(value) ? value.join('\r\n') : value;
-          await copy(copyValue);
-          onAction?.('copy');
-        }}
-        data-testid="databutton-copy"
-        className="-my-3 -me-4 p-3 rounded text-grey-500 bg-transparent border border-transparent hover:bg-grey-100 active:bg-grey-200 focus:outline focus:outline-2 focus:outline-offset-2 focus:outline-blue-500 focus:bg-grey-50"
+    <>
```

Comment:
Do we need this fragment?

Reply:
We do not, this is an artifact from a trying a different way to add the tooltip. Will remove!